Samples/AppLifecycle/Activation/cs-winforms-unpackaged/CsWinFormsActivation/Program.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Windows.Forms;
+using Microsoft.Windows.ApplicationModel.DynamicDependency;
```

Comment:
`using global::Microsoft...`

to avoid ambiguity if implicitUsings are enabled